src/components/library/library.css

```diff
@@ -20,7 +20,7 @@
 
 .library-scroll-grid {
     display: flex;
-    justify-content: flex-start;
+    justify-content: center;
```

Comment:
This gets us part of the way there. This center aligns the items in the container element, but also aligns any items at the bottom of the library to the center as well.

### Before
![image](https://user-images.githubusercontent.com/747641/35336034-1fe11592-00e5-11e8-82c6-e30206e0444b.png)

### After
![image](https://user-images.githubusercontent.com/747641/35336062-349b1eba-00e5-11e8-99e3-d3780e9a01e5.png)

We want to have the items be centered in the space, but any "non-full" rows should be left aligned:
https://stackoverflow.com/questions/32802202/how-to-center-a-flex-container-but-left-align-flex-items